CHANGELOG.md

```diff
@@ -3,6 +3,160 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 4.0.0-alpha.38
+
+_Jul 23, 2021_
+
+Big thanks to the 6 contributors who made this release possible. Here are some highlights ✨:
+
+- 🚀 Introduce row editing feature (#2098) @m4theushw
+- ⚛️ Remove unused row CSS classes (#2327) @ZeeshanTamboli
+- ⚡️ Rename `XGrid` to `DataGridPro` (#2382) @m4theushw
+- ✨ Move packages to `@mui` scope (#2341) @oliviertassinari
+- 💡 Remove `apiRef` from all `options` callback params (#2312) @DanailH
+
+This is the last alpha release. We are moving to beta in the next release, next week.
+
+### `@mui/x-data-grid@v4.0.0-alpha.38` / `@mui/x-data-grid-pro@v4.0.0-alpha.38`
+
+#### Breaking changes
+
+- [DataGrid] Remove unused row CSS classes (#2327) @ZeeshanTamboli
+  The CSS classes `.Mui-odd` and `.Mui-even` were removed from the row. Use the `:nth-child(odd|even)` selector to replace them.
+
+- [DataGridPro] Rename `XGrid` to `DataGridPro` (#2382) @m4theushw
+
+  ```diff
+  - import { XGrid } from '@mui/x-data-grid-pro';
+  + import { DataGridPro } from '@mui/x-data-grid-pro';
```

Comment:
We've been using only one space between the sign and the modified line.

```suggestion
  -import { XGrid } from '@mui/x-data-grid-pro';
  +import { DataGridPro } from '@mui/x-data-grid-pro';
```

x the other diffs.